app/modules/annotations/models.py

```diff
@@ -187,6 +185,96 @@ def get_asset_src(self):
             assset_src = self.asset.src
         return assset_src
 
+    def get_matching_set(self, query=None, load=True):
+        if not self.encounter_guid:
+            raise ValueError(f'{self} has no Encounter so cannot be matched against')
+        if not query or not isinstance(query, dict):
+            query = self.get_matching_set_default_query()
+        log.info(f'finding matching set for {self} using query {query}')
+        return self.elasticsearch(query, load=load)
+
+    def get_matching_set_default_query(self):
+        # n.b. default will not take any locationId or ownership into consideration
+        parts = {'filter': []}
+        enc, enc_edm, sight_edm = self.get_related_extended_data()
```

Comment:
hasn't this function gone now?

Reply:
work in progress.  same with caching.  :)   as the commit message says:
> new way to get edm-data on FeatherModel objects (currently cache-less)